server/server.go

```diff
@@ -256,3 +262,182 @@ func (r *rodeServer) ListOccurrences(ctx context.Context, occurrenceRequest *pb.
 		NextPageToken: "",
 	}, nil
 }
+
+func (r *rodeServer) CreatePolicy(ctx context.Context, policyEntity *pb.PolicyEntity) (*pb.Policy, error) {
+	// TODO maybe check if it already exists (if we think a unique name is required)
+
+	log := r.logger.Named("CreatePolicy")
+
+	policy := &pb.Policy{
+		Id:      uuid.New().String(),
+		Version: 1,
+		Policy:  policyEntity,
+	}
+	str, err := protojson.MarshalOptions{EmitUnpopulated: true}.Marshal(proto.MessageV2(policy))
+	if err != nil {
+		return nil, createError(log, fmt.Sprintf("error marshalling %T to json", policy), err)
+	}
+	res, err := r.esClient.Index(
```

Comment:
We could potentially do `r.es.client.Create` here and set the policy id as the document id as the policy id. That might make get/delete/update a little easier, since we don't need to do a search.